src/main/java/uk/ac/ebi/subs/data/submittable/Sample.java

```diff
@@ -5,6 +5,12 @@
 import uk.ac.ebi.subs.data.component.SampleExternalReference;
 import uk.ac.ebi.subs.data.component.SampleRef;
 import uk.ac.ebi.subs.data.component.SampleRelationship;
+import uk.ac.ebi.subs.data.submittable.sample.msi.Database;
```

Comment:
I think we don't need to add `msi` into the package name.

Reply:
updated.